railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -547,6 +547,28 @@ def run_after_bundle_callbacks
         @after_bundle_callbacks.each(&:call)
       end
 
+      def show_next_steps_banner
+        return if options[:quiet]
+
+        puts <<~NEXTSTEP
+
+          Rails project created and dependencies installed.
+
+          Now switch to the directory where the Rails app was created:
```

Comment:
This is great! I wonder if this should be behind a conditional in case they're already in the Rails app directory. For example, I typically create the directory first, then cd into it, then run `rails new .`